content/en/docs/tasks/policy-enforcement/rate-limit/index.md

```diff
@@ -0,0 +1,359 @@
+---
+title: Enabling Rate Limits using Envoy
+description: This task shows you how to use Istio to dynamically limit the traffic to a service.
+weight: 10
+keywords: [policies,quotas]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: n/a
+---
+
+This task shows you how to use Envoy's native rate limiting in Istio to dynamically limit the traffic to a
+service. In this task, we will apply a global rate-limit for the `productpage` service through ingress gateway that allows
+1 requests per minute across all instances of the service. Additionally, we will apply a local rate-limit for each
+individual `productpage` instance that will allow 10 requests per minute. In this way, we will ensure that our `productpage`
+service handles a maximum of 1 requests per minute through instance gateway, but each `productpage` instance can handle
+up to 10 requests per minute allowing for any in-mesh traffic.
+
+## Before you begin
+
+1. Setup Istio in a Kubernetes cluster by following the instructions in the
+   [Installation Guide](/docs/setup/getting-started/).
+
+1. Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application.
+
+    The Bookinfo sample deploys 3 versions of the `reviews` service:
+
+    * Version v1 doesn’t call the `ratings` service.
+    * Version v2 calls the `ratings` service, and displays each rating as 1 to 5 black stars.
+    * Version v3 calls the `ratings` service, and displays each rating as 1 to 5 red stars.
```

Comment:
Why are you explaining this, given that it has no bearing on this task? All you're demonstrating is how to rate limit calls to productpage, so the versions of reviews are irrelevant.

```suggestion
```